darts/tests/test_filters.py

```diff
@@ -62,7 +64,53 @@ def test_moving_average_multivariate(self):
         self.assertGreater(np.mean(np.abs(ts.values()[:, 1])), np.mean(np.abs(ts_filtered.values()[:, 1])))
 
 
+class GaussianProcessFilterTestCase(DartsBaseTestClass):
+
+    def test_gaussian_process(self):
```

Comment:
Could you maybe make these tests deterministic by fixing the random seed?

Reply:
Good point, done!